src/api/fetchAPI_json_server.ts

```diff
@@ -0,0 +1,31 @@
+import { fetchApi } from "./fetchAPI";
+
+export class fetchAPI_json_server implements fetchApi {
+    url: string;
+    
+    constructor(url: string) {
+        this.url = url;
+    }
+
+    getWorkflows = async () => {
+        const response = await fetch(`${this.url}/workflows`)
+        const data: any = await response.json();
+        return data;
+    };
+    
+    
+    getWorkflow = async (name: string) => {
+        const response = await fetch(`${this.url}/workflow?name=${name}`)
```

Comment:
Why await? It's better to map the Promise value and return the Promise without await and async:
```
return fetch(`${this.url}/workflow?name=${name}`)
.then(response => response.json();
```

Reply:
Does it come down to [writing style](https://medium.com/@dannysoto/should-i-use-async-await-instead-of-just-fetch-in-javascript-710a72731e29), or is there another reason? Anyway it'll be more consistent with previous work 👍